Support-Client/Podfile

```diff
@@ -0,0 +1,19 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Support-Client' do
```

Comment:
can we rename to `SupportClient`, i think its more conventional than having a `-` in the project name